org.mozilla.Firefox.BaseApp.json

```diff
@@ -3,8 +3,35 @@
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "19.08",
     "sdk": "org.freedesktop.Sdk",
-    "separate-locales": false,
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
```

Comment:
How do you expect Firefox to build without these

Reply:
This baseapp won't be used to build firefox. It's needed only at runtime.

Reply:
Lame.

Reply:
From Mozilla POV it's way easier to build things once and then package it to various formats like tar, deb, snap and flatpak.

From my experience, building ff with flatpak requires various quirks and adjustments with every major version update which happens once in couple of weeks unless every dep is vendored including rust and node and that defeats the point of building it exclusively in first place.

I'm afraid that if building in flatpak was requirement then official ff flatpak could never happen.